src/PackageCompiler.jl

```diff
@@ -432,11 +509,13 @@ function create_sysimage(packages::Union{Symbol, Vector{Symbol}}=Symbol[];
         end
         base_sysimage = create_fresh_base_sysimage(sysimage_stdlibs; cpu_target=cpu_target)
     else
-        if base_sysimage == nothing
+        if base_sysimage === nothing
```

Comment:
Is using `isnothing()` more Julian?

Reply:
I'm not sure, but I made this change.  Thanks for reviewing, @Byrth!